auto_install/install.sh

```diff
@@ -2597,7 +2792,8 @@ All incomplete posts or bug reports will be ignored or deleted.\\n\\nThank you f
 		whiptail --title "Rebooting" --msgbox "The system will now reboot." ${r} ${c}
 		printf "\\nRebooting system...\\n"
 		$SUDO sleep 3
-		$SUDO shutdown -r now
+
+		"${SUDO}" reboot
```

Comment:
no need for extra empty line

Even tho nowadays reboot is widely accepted in any distro release and unless the standard has changed, `shutdown -r now` should be the correct way.

Reply:
The problem was that `shutdown -r now` doesn't work with Alpine Linux, so if it is preferred, the condition of the initial commit would need to be recovered. But  in all Debian variants (systemd as well as SysV), `reboot` should be safe to use.

Reply:
well that's good enough. we can keep the reboot.